app/src/main/java/com/nilhcem/blenamebadge/ui/fragments/main_text/MainTextDrawableNavigator.kt

```diff
@@ -1,35 +0,0 @@
-package com.nilhcem.blenamebadge.ui.fragments.main_text
-
-import android.widget.LinearLayout
-import android.widget.TextView
-import com.nilhcem.blenamebadge.data.DrawableInfo
-import com.nilhcem.blenamebadge.data.device.model.DataToSend
-import pl.droidsonroids.gif.GifImageView
-
-interface MainTextDrawableNavigator {
```

Comment:
When I was learning mvp, I had an impression of creating interfaces for every class. @iamareebjamal sir would be the perfect person to answer this

Reply:
We are not following MVP

Reply:
@iamareebjamal I wrote MVP instead of MVVM 😂. My bad

Reply:
There are no interfaces in MVVM

Reply:
Great, @anhanh11001. This is valid then